ios/RNSScreenStack.mm

```diff
@@ -1246,13 +1242,13 @@ - (void)prepareForRecycle
 
 - (void)invalidate
 {
-  _invalidated = YES;
-  for (UIViewController *controller in _presentedModals) {
-    [controller dismissViewControllerAnimated:NO completion:nil];
-  }
-  [_presentedModals removeAllObjects];
-  [_controller willMoveToParentViewController:nil];
-  [_controller removeFromParentViewController];
+  //  _invalidated = YES;
+  //  for (UIViewController *controller in _presentedModals) {
+  //    [controller dismissViewControllerAnimated:NO completion:nil];
+  //  }
+  //  [_presentedModals removeAllObjects];
+  //  [_controller willMoveToParentViewController:nil];
+  //  [_controller removeFromParentViewController];
```

Comment:
I don't think this should be commented 😄

Reply:
You're totally right, I didn't notice those changes being pushed 👍

Reply:
https://github.com/software-mansion/react-native-screens/pull/2188/commits/35873033b1858a7bfa5cd606440c9441f6879b99